rllib/core/rl_module/rl_module.py

```diff
@@ -237,6 +237,8 @@ def from_model_config(
         """
         raise NotImplementedError
 
+    # TODO: (Artur) Add a method `from_catalog` that creates RLModule from Catalog
```

Comment:
@kouroshHakha This is something you "wished" for a couple days back, that we don't need just right now. Still added this todo.